stable/ark/templates/hook-delete.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: delete-ark-config
+  labels:
+    chart: {{ template "ark.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    app: {{ template "ark.name" . }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-weight": "2"
+spec:
+  template:
+    metadata:
+      name: delete-ark-config
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: delete-ark-config
+          image: {{ required "A docker image with kubectl" .Values.kubectl.image.repository }}/{{ required "A docker image with kubectl" .Values.kubectl.image.tag }}
+          imagePullPolicy: Always
+          command: ["kubectl", "delete", "-f", "/tmp/"]
```

Comment:
If the image has entry point `kubectl` or command `kubectl`,  you can pass just the arguments, without the `kubectl` such as here: https://hub.docker.com/r/lachlanevenson/k8s-kubectl/ imo.